.vscode/launch.json

```diff
@@ -22,7 +22,7 @@
       "name": "Debug Jest e2e",
```

Comment:
i think  .vscode/ files should not be in a project

Reply:
hmm, `.gitignore` allows to be:
```
# IDE - VSCode
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```

Reply:
should we comment following lines? 

https://github.com/lidofinance/lido-keys-api/blob/6f2d99c66816ff393d54abd1f4bb646a889271a4/.gitignore#L34-L39